controllers/articles.controller.js

```diff
@@ -65,3 +66,12 @@ exports.patchArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeComment(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
res.sendStatus(204) can be used as an alternative when only sending back a status. Just an fyi, one isn't generally considered more preferable to the other in my experience!